cmd/cfgfiles/tapir-pop.yaml

```
@@ -0,0 +1,66 @@
+dnsengine:
+    addresses: [ ### EDIT Addresses (ip:port) to listen to for RPZ XFR requests ]
+    active: true
+    name: TAPIR-POP DNS Engine
+    logfile: /var/log/dnstapir/pop-dnsengine.log
+
+cli:
+    tapir-pop:
+        url: https://127.0.0.1:9099/api/v1
```

YAMLlint (1.37.1)

[warning] cannot infer indentation: unexpected token

(indentation)